ja/news/_posts/2024-10-07-ruby-3-4-0-preview2-released.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: news_post
+title: "Ruby 3.4.0 preview2 リリース"
+author: "naruse"
+translator: "motohiro-mm"
+date: 2024-10-07 00:00:00 +0000
+lang: ja
+---
+
+{% assign release = site.data.releases | where: "version", "3.4.0-preview2" | first %}
+Ruby {{ release.version }}がリリースされました。
+
+## Prism
+
+デフォルトのパーサーを parse.y から Prism に変更しました。　[[Feature #20564]]
+
+## 言語機能の変更
+
+* `frozen_string_literal`のコメントがないファイルで文字列リテラルが凍結されたように振る舞うようになりました。
+  文字列リテラルが破壊的に変更された場合、非推奨の警告が表示されます。
+  この警告は `-W:deprecated` または `Warning[:deprecated] = true` で有効にすることができます。
+  コマンドライン引数で`--disable-frozen-string-literal` を指定してRubyを実行すると、この変更を無効にできます。 [[Feature #20205]]
+
+* `it`がブロックパラメータを参照するために追加されました。 [[Feature #18980]]
+
+* メソッド呼び出し時のnilのキーワードスプラットが使えるようになりました。
+  `**nil`は`**{}`と同様に扱われ、キーワードは渡されず、変換メソッドも呼び出されません。 [[Bug #20064]]
+
+* インデックスにブロックを渡せなくなりました。 [[Bug #19918]]
+
+* インデックスにキーワード引数が使えなくなりました。 [[Bug #20218]]
+
+## コアクラスの更新
+注：特に重要なクラスアップデートのみを掲載しています。
+
+* Exception
+
+  * Exception#set_backtraceが`Thread::Backtrace::Location`の配列を受け付けるようになりました。
+    `Kernel#raise`と`Thread#raise`、`Fiber#raise`も同様に新しいフォーマットを受け付けます。[[Feature #13557]]
+
+* Range
+
+  * rangeが列挙可能でない場合、`Range#size`がTypeErrorを発生させるようになりました。[[Misc #18984]]
+
+
+## 互換性に関する変更
+
+注：バグフィックスは掲載していません。
+
+* エラーメッセージとバックトレースの表示が変更されました。
+  * 冒頭の引用符にはバッククォートの代わりにシングルクォートを使用します。 [[Feature #16495]]
+  * メソッド名の前にクラス名を表示します(クラスが永続的な名前を持つ場合のみ)。 [[Feature #19117]]
+  * `Kernel#caller`、`Thread::Backtrace::Location`のメソッドなどがそれに応じて変更されました。
+
+  ```
+  Old:
+  test.rb:1:in `foo': undefined method `time' for an instance of Integer
+          from test.rb:2:in `<main>'
```

Comment:
```suggestion
          from test.rb:2:in '<main>'
```

👉🏻 https://github.com/ruby/www.ruby-lang.org/pull/3384

Reply:
https://github.com/ruby/www.ruby-lang.org/pull/3386#discussion_r1792846268
I had the same result as this.

Reply:
Sorry my fault on this one!